common.php

```diff
@@ -210,7 +210,10 @@
         ),
         'default_country' => 'Australia',
         'canViewSummary' => true,
+        'gp_label' => 'Referring Practitioner',
+        'general_practitioner_label' => "Referring Practitioner",
```

Comment:
This attribute already exists in line 201, its value just needed to be changed to "Referring Practitioner"

Reply:
I've changed it now